### PR TITLE
Wip util jline 2m

### PR DESCRIPTION
**Pull request by raboof**
(Continuation of #1)

### CODE REVIEW

**Comment by 2m** [comment]
I'll check if this works for sbt-dependency-graph.

**Comment by 2m** [comment]
Yup. Seems all good!

File: src/main/scala-sbt-0.13/compat.scala [hunk 1]
```diff
@@ -89,6 +89,14 @@ package internal {
     object `package`
   }
 
+  package util {
+    object `package` {
```
Comment (dwijnand) [inline src/main/scala-sbt-0.13/compat.scala]: Lose the package object, just declare JLine in the util package.

File: src/main/scala-sbt-0.13/compat.scala [hunk 2]
```diff
@@ -89,6 +89,14 @@ package internal {
...
+      object JLine {
+        val usingTerminal = sbt.JLine.usingTerminal _
```
Comment (dwijnand) [inline src/main/scala-sbt-0.13/compat.scala]: Also I'm not sure if we should do this or not.. it's cool, but maybe we should just stick to declaring it as:

```scala
def usingTerminal[T](f: jline.Terminal => T): T
```

**Comment by raboof** [comment]
Made those 2 changes - sorry for taking a while :)

**Comment by dwijnand** [comment]
v1.1.0 tag cut, release in progress: https://travis-ci.org/dwijnand/sbt-compat/builds/284099230.

**Pull request merged**